packages/amesos2/src/Amesos2_Tacho_decl.hpp

```diff
@@ -96,10 +98,20 @@ class TachoSolver : public SolverCore<Amesos2::TachoSolver, Matrix, Vector>
   typedef Tacho::ordinal_type                                  ordinal_type;
   typedef Tacho::size_type                                        size_type;
   typedef Kokkos::DefaultHostExecutionSpace                   HostSpaceType;
-  typedef Kokkos::TaskScheduler<HostSpaceType>                SchedulerType;
-  typedef Kokkos::View<size_type*,HostSpaceType>            size_type_array;
-  typedef Kokkos::View<ordinal_type*,HostSpaceType>      ordinal_type_array;
-  typedef Kokkos::View<tacho_type*, HostSpaceType>         value_type_array;
+
+  typedef Kokkos::DefaultExecutionSpace                      DeviceSpaceType;
+
```

Comment:
Why do you name execution space as device space type ?

Reply:
The naming was carry-over from the original code before the PR. I switched to exec_space_type_ and host_exec_space_type.